source/apiVolontaria/volunteer/models.py

```diff
@@ -227,6 +240,18 @@ class Meta:
     event = models.ForeignKey(Event, related_name='participation')
     user = models.ForeignKey(User, related_name='participation')
 
+    presence_duration_minutes = models.PositiveIntegerField(
+        default=0,
```

Comment:
By default it's `null` and not `0`, we don't know how many time the volunteer came to the event.